services/api/routers/crud/crud.py

```diff
@@ -32,7 +32,7 @@ def get_closest_station(zipcode: str, db: Session):
     loc = zipcode_to_latlong(zipcode)
     stmt = text(
         """
-        SELECT station_name, latitude, longitude, location_coord
+        SELECT station_id, station_name, latitude, longitude, location_coord
```

Comment:
hmm sooo I took station_id out of the readings table with my latest changes, replaced it with full_aqs_id....I think we'll be switching to the composite tables shortly anywhoo, but maybe we use full_aqs_id here for now?